preprocessing/preprocessing_one_hot_encoding.py

```diff
@@ -0,0 +1,119 @@
+import pandas as pd
```

Comment:
File appears as new because I corrected a typo in the file name, you may need to change that in your notebook

Reply:
Everything fine from my perspective